lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -151,8 +151,37 @@ def which(cmd):
     return None
 
 
-class MPI(object):
+class RunFileLoopMPI(RunFile, RunLoop):
+    """
+    RunFile specific for MPI because intel doesn't feel like fixing their shit
```

Comment:
yeah, maybe be less aggressive here.. ;)

`
"""
Combination of RunFile and RunLoop to support output to file,
while also checking whether any output has been produced after a specified amount of time.
"""